recipes-crosswalk/crosswalk/crosswalk/Ostro-OS-XT-Fallback-MADV_FREE-to-MADV_DONTNEED.patch

```diff
@@ -0,0 +1,33 @@
+From 8d38f1dbac1cb836bfaaf9d05152b38dfed93535 Mon Sep 17 00:00:00 2001
+From: Ningxin Hu <ningxin@intel.com>
+Date: Fri, 9 Sep 2016 13:18:05 +0000
+Subject: [PATCH] [Ostro OS XT] Fallback MADV_FREE to MADV_DONTNEED
+
+MADV_FREE has wrong value on Ostro OS XT which causes madvise always
+returns -1. Fallback to MADV_DONTNEED.
+
+TODO=fix in Ostro OS XT upstream.
+---
+ third_party/WebKit/Source/wtf/PageAllocator.cpp | 6 +++++-
+ 1 file changed, 5 insertions(+), 1 deletion(-)
+
+diff --git a/third_party/WebKit/Source/wtf/PageAllocator.cpp b/third_party/WebKit/Source/wtf/PageAllocator.cpp
+index 7f7a884..4f5a504 100644
+--- a/third_party/WebKit/Source/wtf/PageAllocator.cpp
++++ b/third_party/WebKit/Source/wtf/PageAllocator.cpp
+@@ -40,7 +40,11 @@
+ #include <errno.h>
+ #include <sys/mman.h>
+ 
+-#ifndef MADV_FREE
++// [Ostro OS XT] MADV_FREE has wrong value which causes madvise
++// always returns -1. Fallback to MADV_DONTNEED.
++// TODO: fix in Ostro OS XT upstream.
++#ifdef MADV_FREE
++#undef MADV_FREE
+ #define MADV_FREE MADV_DONTNEED
```

Comment:
Can we move this line after `#endif`? Otherwise `MADV_FREE` will be undefined if it is not already defined.